spec/models/aged_offspring_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'rails_helper'
+include Warden::Test::Helpers
+
+RSpec.describe AgedOffspring, type: :model do
+  active_record_offspring = 'activerecord.errors.models.offspring.attributes'
+
+  context "when authenticated as admin" do
+    let(:admin) { FactoryGirl.create(:admin) }
+
```

Comment:
You don't need to have admin here if you don't log in... You are testing a model, not a request